htdocs/core/modules/mailings/fraise.modules.php

```diff
@@ -203,7 +203,7 @@ function add_to_target($mailing_id,$filtersarray=array())
         $sql.= " a.lastname, a.firstname,";
         $sql.= " a.datefin, a.civility as civility_id, a.login, a.societe";    // Other fields
         $sql.= " FROM ".MAIN_DB_PREFIX."adherent as a, ".MAIN_DB_PREFIX."adherent_type as ta";
-        $sql.= " WHERE a.email <> ''";     // Note that null != '' is false
+        $sql.= " WHERE a.email <> '' AND a.entity IN (".getEntity('member').")";     // Note that null != '' is false
```

Comment:
hello @ptibogxiv 

it's better to filter with entity before all for mysql performance
`$sql.= " WHERE a.entity IN (".getEntity('member').") AND a.email <> ''";`